l10n/en/firefox/browsers/mobile/ios-summarizer.ftl

```diff
@@ -9,42 +9,33 @@
 # that conveys energy, freshness, or innovation, rather than a literal "shake" action.
 mobile-ios-summarizer-page-title = Ready to shake things up?
 mobile-ios-summarizer-page-description =
-    Get instant, tailored summaries in { -brand-name-firefox } { -brand-name-ios }.
+    Get instant, tailored summaries in { -brand-name-firefox } for iOS.
     Shake your phone to cut through the clutter with recipe ingredients, sports highlights,
     or article overviews in seconds.
 
-# Note for localizers: "shake things up" is an English idiom meaning to cause change, disrupt the usual way,
-# or bring excitement. Please translate with an equivalent expression in your language
-# that conveys energy, freshness, or innovation, rather than a literal "shake" action.
-mobile-ios-summarizer-hero-title = Ready to shake things up?
-mobile-ios-summarizer-hero-subtitle = Instant summaries in { -brand-name-firefox } { -brand-name-ios }
-# Note for localizers: This is used under a QR code image that redirects to downloading the app in the app store.
+mobile-ios-summarizer-hero-subtitle = Instant summaries in { -brand-name-firefox } for iOS
 # Note for localizers: This is used in a link that redirects to downloading the app in the app store when clicked.
-mobile-ios-summarizer-cta-get-app = Get the app
+mobile-ios-summarizer-cta-get-app = Get { -brand-name-firefox } for iOS
+# Note for localizers: This is used under a QR code image that redirects to downloading the app in the app store.
 mobile-ios-summarizer-hero-qr-caption = Scan to download the app!
 
-# Note for localizers:
-# "Bloated with fluff" in this context means pages overloaded with filler content, ads, or other useless content.
-# Use a natural equivalent, not literal wording.
-mobile-ios-summarizer-pitch1-title = Tired of endless web pages bloated with fluff?
+mobile-ios-summarizer-pitch1-title = Cut through the clutter
 mobile-ios-summarizer-pitch1-body =
-    Struggling to skim recipes buried in stories or news articles that waste your time?
-    In our chaotic digital world, who wants to feel overwhelmed and inefficient?
-    You’re not alone, but { -brand-name-firefox } is your guide to smarter browsing.
+    Web pages are full of filler, with recipes buried under endless scrolling and news that takes too long to get to the point.
```

Comment:
```suggestion
    Web pages are full of filler, with recipes buried under endless scrolling and news that take too long to get to the point.
```

filler or fillers?

Reply:
It's filler in the design. Let me double check with content.